test/runtests.jl

```diff
@@ -57,12 +57,21 @@ out2, _ = ZigZagBoomerang.pdmp(∇ϕhat, x0, θ0, T, 4.0, B)
 
 end
 
+
+x0, θ0 = 1.41, +0.1
+B = Boomerang1d(1.8, 1.2, 2.5)
+out2, _ = ZigZagBoomerang.pdmp(∇ϕ, x0, θ0, T, 20.0, B)
+
+
 @testset "Boomerang1d" begin
     @test T/10 < length(out2) < T*10
     dt = 0.01
     traj = ZigZagBoomerang.discretization(out2, B, dt)
     est = mean(traj.x)
-    @test abs(est-pi) < 3/sqrt(length(out2))
+    @test abs(est - pi/2) < 5/sqrt(length(out2) - T*2.5)
+    est2 = var(traj.x)
+    @test abs(est2 - 1.5) < 8/sqrt(length(out2) - T*2.5)
```

Comment:
Doesn't work quite so good as ZigZag here. I am excluding the refreshment events in determining the sample size.